src/js/index.js

```diff
@@ -101,11 +90,34 @@ export function getElementsBondsData(element1, element2, order = undefined) {
  * @returns {Number}
  */
 export function getElementAtomicRadius(elementSymbol) {
-    const elementName = Object.keys(PERIODIC_TABLE).find(
-        (key) => elementSymbol === PERIODIC_TABLE[key].symbol,
-    );
-    if (elementName) {
-        return PERIODIC_TABLE[elementName].atomic_radius_pm * 0.01;
+    const config = PERIODIC_TABLE[elementSymbol];
+    if (config) {
+        return PERIODIC_TABLE[elementSymbol].atomic_radius_pm / 100;
     }
     return 1.0;
 }
+
+/**
+ *
+ * @param elements {String[]} - List of elements (as symbols)
+ * @param properties {String[]} - List of properties
+ * @param propMap {Object} - Object mapping property names to custom names (e.g. {'atomic_radius_pm': 'radius'})
+ * @param separator {String} - Separator string for the suffix (default: ':')
+ * @returns {Object[]} - List of Objects {property_key: property_value}
```

Comment:
It would probably be good to provide an example of what this returns, for clarity.

Reply:
See https://github.com/Exabyte-io/express/blob/a182f7dccfc5c715aa4ca4adf0c20be85f90f7b7/express/parsers/mixins/ionic.py for reference

Reply:
Example input and output were added to the docstring.